pkg/gui/context/stash_context.go

```diff
@@ -70,3 +70,12 @@ func (self *StashContext) GetSelectedRefName() string {
 
 	return item.RefName()
 }
+
+func (self *StashContext) GetParentRefName() string {
+	item := self.GetSelected()
+	if item == nil {
+		return ""
+	}
+
+	return item.RefName() + "^"
```

Comment:
what are your thoughts on stash entries also having a ParentRefName method to make this consistent? That would allow the stash to satisfy the Ref interface described above